### ISSUE TITLE
AutoCrystal have some problem

### ISSUE THREAD

**Issue by dianwangwin** [bebeli555/CookieClient#10]
It wont let Crystal broken when you place it sometimes in 2b2t 
Please Fix it :-)

**Comment by bebeli555** [comment]
That happens if u have the ticsk setting too low. Like the default 2 is too low for 2b2t for me 3-4 ticks seems to work. But basically if u have it too low then 2b2ts anticheat will flag you and you can no longer break them so set the ticks setting to like 3 or 4
